pkg/providers/amifamily/ami.go

```diff
@@ -41,7 +41,7 @@ import (
 )
 
 type Provider interface {
-	Get(ctx context.Context, nodeClass *v1beta1.EC2NodeClass, options *Options) (AMIs, error)
+	List(ctx context.Context, nodeClass *v1beta1.EC2NodeClass, options *Options) (AMIs, error)
```

Comment:
```suggestion
	List(ctx context.Context, nodeClass *v1beta1.EC2NodeClass) (AMIs, error)
```

Another question: I think we can do this. We don't actually use the passed-through options anywhere.